package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.7.12",
+    "@mate-academy/scripts": "^1.2.8",
     "eslint": "^5.16.0",
     "eslint-plugin-jest": "^22.4.1",
```

Comment:
You need to commit only those files that you changed manually
> git add src/<task_name>.js

Check this out [the guideline](https://github.com/mate-academy/js_task-guideline/blob/master/README.md)